tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-product-filter-test.php

```
@@ -0,0 +1,1117 @@
+		$filter     = new WC_Stripe_Agentic_Commerce_Product_Filter();
+		$query_args = $filter->get_query_args();
+
+		$query_args['return'] = 'ids';
+
+		$query_results = wc_get_products( $query_args );
+
+		if ( $should_include ) {
+			$this->assertEquals( $product_ids_to_query, $query_results );
+		} else {
+			$this->assertSame( [], $query_results );
+		}
```

PHPMD (2.15.0)

[error] The method test_product_id_query_handles_product_type uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)